src/main/java/org/broadinstitute/hellbender/tools/copynumber/utils/annotatedregion/SimpleAnnotatedGenomicRegion.java

```diff
@@ -113,35 +144,44 @@ protected SimpleAnnotatedGenomicRegion createRecord(final DataLine dataLine) {
     }
 
     /**
-     *  Write a tsv file of annotated regions with the specified strings for the position headers.
+     *  This method will likely be removed in future releases.
+     *
+     *  Reads entire TSV file in one command and stores in RAM.  Please see {@link SimpleAnnotatedGenomicRegion::CONTIG_HEADER},
+     *  {@link SimpleAnnotatedGenomicRegion::START_HEADER}, and
+     *  {@link SimpleAnnotatedGenomicRegion::END_HEADER} for defining the genomic region.
      *
-     * @param regions list of locatables and associated annotations for writing.  One per line.
-     * @param outputFile location of the output file.
+     * @param tsvRegionFile -- File containing tsv of genomic regions and annotations per line.  E.g. a segment file.
+     * @param headersOfInterest -- The columns that should be in the resulting annotated region.  This list should
+     *                          not include any headers that are used to define the region (e.g. contig, start, end)
+     * @return annotated regions with one line in the input file for each entry of the list.  Never {<@UNVERIFIED|@code> null}
      */
-    public static void writeAnnotatedRegionsAsTsv(final List<SimpleAnnotatedGenomicRegion> regions, final File outputFile) {
-
-        // Can't do Arrays.asList, since we will be adding to this list.
-        final List<String> columnHeaders = Lists.newArrayList(CONTIG_HEADER, START_HEADER, END_HEADER);
-        final List<String> otherHeaders = new ArrayList<>(regions.get(0).getAnnotations().keySet());
-        otherHeaders.sort(String::compareTo);
+    public static List<SimpleAnnotatedGenomicRegion> readAnnotatedRegions(final File tsvRegionFile, final Set<String> headersOfInterest) {
```

Comment:
Same here (and this is essentially only used in test code, as well).

Reply:
This method was deleted.
Done